### DISCUSSION TITLE
Is there any way to make 'set textwidth=80' work?

### DISCUSSION THREAD

**Discussion by gaoqiangks** [vscode-neovim/vscode-neovim#1324] [Q&A]
I want to automatically break a long line, which is done by `set textwidth=80` in vim.  However, in vscode-neovim, it seems that all things related to insert mode are done by vscode, and insert-mode settings in neovim are ignored.  So, is there any way to realize this?

**Comment by maurocig** [comment]
I just use prettier. you can set {"printWidth: 80"} in .prettierrc file

**Comment by theol0403** [comment]
This has to be done on vscode side, probably with formatting extension.
